### PR TITLE
MM-31674 Cloud app/product notices for ext dependecies

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: mattermost/mattermost-server#16871

Updated:
- Cloud Admin Guide > Configuration > Site Configuration > In-Product Notices > Administrator Notices
   - Updated description to include deprecation of external dependencies

### CODE REVIEW

**Comment by cwarnermm** [comment]
@wiersgallak - Please confirm that Admin Notices will inform Cloud System Admins of the following:
- when a new server version is available
- when external dependencies are being deprecated
- when a server upgrade is recommended due to ending support life cycles
- when recommended server configuration options can optimize the user experience of their deployment

I suspect that not all of these are true for Cloud System Admins; I can update the product documentation accordingly.

**Review by justinegeffen — APPROVED** [review]

**Comment by wiersgallak** [comment]
@cwarnermm We will not use in-product notices for Cloud for these use cases as they are on-premise only: 
- when a new server version is available
- when external dependencies are being deprecated
- when a server upgrade is recommended due to ending support life cycles

This one may still be applicable as long as the configuration option is available in the System Console for Cloud instances: 
- when recommended server configuration options can optimize the user experience of their deployment

File: source/cloud/cloud-administration/site-configuration.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Mattermost in-product notices keep users and administrators informed of the newe
 Administrator Notices
 ^^^^^^^^^^^^^^^^^^^^^
 
-Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
+Administrator notices inform System Admins when a new server version is available, when external dependencies are being deprecated, or when a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
```
Comment (wiersgallak) [inline source/cloud/cloud-administration/site-configuration.rst:35]:
```suggestion
System Admins may receive notices about recommended configuration options to optimize the user experience of their workspace.
```

```

**Review by wiersgallak — CHANGES_REQUESTED** [review]
Cloud will not have external dependencies or life cycles that admins need to worry about.  We manage all of that for them.

**Comment by cwarnermm** [comment]
Thanks, @wiersgallak!

**Review by wiersgallak — APPROVED** [review]

**Pull request merged**